Resources/Prototypes/Nyanotrasen/Entities/Mobs/Player/moth.yml

```diff
@@ -25,9 +25,10 @@
       types:
         Asphyxiation: -1.0
   - type: Vocal # these are all the same
-    maleScream: "/Audio/Voice/Moth/scream_moth.ogg"
-    femaleScream: "/Audio/Voice/Moth/scream_moth.ogg"
-    wilhelm: "/Audio/Voice/Moth/scream_moth.ogg"
```

Comment:
Moths need their wilhelm scream to be their scream, or they'll sound like the default stock wilhelm scream when it happens. The former behavior had all of their screams the same.